patches/chromium/command-ismediakey.patch

```diff
@@ -116,3 +104,23 @@ index f4e3126a4efd66f05c4f13e40ba23db10b8cca96..bb4c1a891dd13855227b39a0e582fd4d
      return event;
    }
  
+@@ -221,19 +230,6 @@ static CGEventRef EventTapCallback(CGEventTapProxy proxy,
+ std::unique_ptr<MediaKeysListener> MediaKeysListener::Create(
+     MediaKeysListener::Delegate* delegate,
+     MediaKeysListener::Scope scope) {
+-  // For Mac OS 10.12.2 or later, we want to use MPRemoteCommandCenter for
```

Comment:
Extensions allow registering shortcuts with `MPRemoteCommandCenter` code path, can you check how they achieve it https://cs.chromium.org/chromium/src/chrome/browser/extensions/extension_keybinding_registry.cc?l=172-202 and avoid the patch if possible.

Reply:
Right - they do, but under the same constraints that i outlined above. All media key options will only work when there is something to play (a Now Playing item) that will trigger the remote receiver; and will similarly not work in the way that we want them to work for users.

Reply:
> Extensions face the same problem that we do,

Was this verified ? As I see the shortcut registration is part of https://developer.chrome.com/extensions/commands which doesn't mention about restrictions to media key accelerators.

I maybe wrong here, but if this behavior is verified for the extensions then I am good with the patch.

Reply:
@deepak1556 I verified the behavior in Chrome proper with the following gist: https://gist.github.com/codebytere/52aad32238c74894b6b4eaa80e40893d.

If you load that as an unpacked extension, open the background page, and then press `MediaPlayPause`, you'll see nothing logged to the background page console. if you open youtube, start a video, and then try again, messages will begin to log to console.

Reply:
Great thanks for verifying, can confirm the behavior.

Style nit: Add this code path behind a buildflag which we can disable instead of deleting in the patch, so that its easier to upstream if that is ever considered.